setup.py

```diff
@@ -42,8 +42,10 @@
     'stylecheck': [
         'autopep8==1.5.5',
         'flake8==3.8.4',
+        'numpy==1.20.*',
```

Comment:
`mypy` seems to require also type hints of `pytest`. May I split these requirements to `typecheck`?
```py
'typecheck': [
    'mypy==0.812',
    'numpy==1.20.*'
    'pytest>=6.2',
]
```

Reply:
Had an offline discussion with @kmaehashi, and decided to install them at `pretest.yml` instead.